packages/features/bookings/lib/handleNewBooking/validateBookingTimeIsNotOutOfBounds.ts

```diff
@@ -52,11 +58,13 @@ export const validateBookingTimeIsNotOutOfBounds = async <T extends ValidateBook
   if (timeOutOfBounds) {
     const error = {
       errorCode: "BookingTimeOutOfBounds",
-      message: `EventType '${eventType.eventName}' cannot be booked at this time.`,
+      message: `EventType '${eventType.title}' cannot be booked at this time. Reason: ${cause}`,
```

Comment:
`eventName` is null by default, so unless there's a custom eventName set by the host, we would see '' or null here in the error message in logs as well as for the booker.

Reply:
I think we could change the message to same 'unavailable_user_errors' so that the user could try again with differnet slot.

Internally we would know what the actual reason could be.

Also, that gave me an idea that we should track the retry of the user and if even second try by the user fails, it is strong signal

Reply:
Yeah I guess we don't need to tell the user the precise reason, it is for us to know.